roles/openshift_logging_fluentd/tasks/main.yaml

```diff
@@ -0,0 +1,225 @@
+        dest: "{{mktemp.stdout}}/fluentd-throttle-config.yaml"
+      when: fluentd_throttle_contents is undefined
+      changed_when: no
+
+    - copy:
+        src: secure-forward.conf
+        dest: "{{mktemp.stdout}}/secure-forward.conf"
+      when: fluentd_securefoward_contents is undefined
+      changed_when: no
+
+    - copy:
+        content: "{{fluentd_config_contents}}"
+        dest: "{{mktemp.stdout}}/fluent.conf"
+      when: fluentd_config_contents is defined
+      changed_when: no
+
+    - copy:
+        content: "{{fluentd_throttle_contents}}"
+        dest: "{{mktemp.stdout}}/fluentd-throttle-config.yaml"
+      when: fluentd_throttle_contents is defined
+      changed_when: no
+
+    - copy:
+        content: "{{fluentd_secureforward_contents}}"
+        dest: "{{mktemp.stdout}}/secure-forward.conf"
+      when: fluentd_secureforward_contents is defined
+      changed_when: no
+
+    - command: >
+        {{ openshift.common.client_binary }} --config={{ mktemp.stdout }}/admin.kubeconfig create configmap logging-fluentd
+        --from-file=fluent.conf={{mktemp.stdout}}/fluent.conf --from-file=throttle-config.yaml={{mktemp.stdout}}/fluentd-throttle-config.yaml
+        --from-file=secure-forward.conf={{mktemp.stdout}}/secure-forward.conf -o yaml --dry-run
+      register: fluentd_configmap
+      changed_when: no
+
+    - copy:
+        content: "{{fluentd_configmap.stdout}}"
+        dest: "{{mktemp.stdout}}/templates/logging-fluentd-configmap.yaml"
+      when: fluentd_configmap.stdout is defined
+      changed_when: no
+  check_mode: no
+
+- name: Set logging-fluentd configmap
+  oc_obj:
+    state: present
+    kubeconfig: {{ mktemp.stdout }}/admin.kubeconfig
+    name: logging-fluentd
+    namespace: {{ opneshift_logging_namespace }}
+    kind: configmap
+    files:
+    - name: logging-fluentd-configmap
+      path: {{ mktemp.stdout }}/templates/logging-fluentd-configmap.yaml
+    delete_after: true
+
+# create Fluentd secret
+- name: Retrieving the cert to use when generating secrets for Fluentd
+  slurp:
+    src: "{{generated_certs_dir}}/{{item.file}}"
+  register: key_pairs
+  with_items:
+    - { name: "ca_file", file: "ca.crt" }
+    - { name: "fluentd_key", file: "system.logging.fluentd.key"}
+    - { name: "fluentd_cert", file: "system.logging.fluentd.crt"}
+
+- name: Set logging-fluentd secret
+  oc_secret:
+    state: present
+    kubeconfig: {{ mktemp.stdout }}/admin.kubeconfig
+    name: logging-fluentd
+    namespace: {{ openshift_logging_namespace }}
+    contents:
+    - path: ca
+      data: "{{key_pairs | entry_from_named_pair('ca_file')| b64decode }}"
+    - path: key
+      data: "{{key_pairs | entry_from_named_pair('fluentd_key')| b64decode }}"
+    - path: cert
+      data: "{{key_pairs | entry_from_named_pair('fluentd_cert')| b64decode }}"
+
+# create Fluentd daemonset
+- name: Generate logging-fluentd daemonset definition
+  template:
+    src: {{ openshift_logging_fluentd_version }}/fluentd.j2
```

Comment:
i had started hashing out logic for versions at the end of the day yesterday. Looks like I used two different variable names. I'll either change this to be `fluentd_version` or update the fact set in `determine_version.yaml` to match the above